weekly-updates/next.md

```diff
@@ -22,7 +22,17 @@ Next weeks call will be on the 3rd October at 1000CEST on [this line](https://zo
 ### [Talks & Articles](https://github.com/solid/information/blob/master/press.md)
 Have you seen any articles or talks about Solid this week? If you've seen any talks, articles, or written blog posts about Solid please do send them over for next week.
 
-* 
+## Solid implementation
+The brand new version 0.9 of **Startin’blox** is out in beta this week !
```

Comment:
People coming across this in TWiS might not be aware of what this is, so a one-line summary to describe the project could be helpful. I yanked the following from the README as a suggestion:

```suggestion
The brand new version 0.9 of **Startin’blox** is out in beta this week! Startin'blox is a collection of Web Components that aim to enable anyone with little development skills to create their own Solid-based web application.
```

Reply:
I agree, @Vinnl. Perhaps you should make your own PR against https://github.com/solid/information to get that in, since your comment was apparently overlooked when @Mitzi-Laszlo merged this PR.